src/constants.js

```diff
@@ -190,6 +190,23 @@ export const EXPLORER_TESTNET_URL = 'https://explorer.testnet.hathor.network/';
 export const EXPLORER_SERVICE_TESTNET_URL = 'https://explorer-service.testnet.hathor.network/';
 export const TX_MINING_SERVICE_TESTNET_URL = 'https://txmining.testnet.hathor.network/';
 
+// Nano testnet settings:
+export const NETWORK_NANO_TESTNET = 'testnet';
+export const NODE_SERVER_NANO_TESTNET_URL = 'https://node1.nano-testnet.hathor.network/v1a/';
+export const EXPLORER_NANO_TESTNET_URL = 'https://explorer.alpha.nano-testnet.hathor.network/';
+export const TX_MINING_SERVICE_NANO_TESTNET_URL = 'https://txmining.nano-testnet.hathor.network/';
+
+export const PRE_SETTINGS_NANO_TESTNET = {
+  stage: STAGE_TESTNET,
+  network: NETWORK_NANO_TESTNET,
+  walletServiceUrl: WALLET_SERVICE_TESTNET_BASE_URL,
+  walletServiceWsUrl: WALLET_SERVICE_TESTNET_BASE_WS_URL,
```

Comment:
Is it the same as our public testnet? I know that we won't allow Hathor Play to connect to wallet service but is it a required key?

Reply:
They're not required, I agree we should remove them, done